test/helpers/position_assertions.cc

```diff
@@ -1187,44 +1205,58 @@ void SymbolSearchAssertion::checkAllForQuery(std::string query,
         auto &location = symbol->location;
         auto line = fmt::format("{}:{:{}}", location->uri, location->range->start->line, lineWidth);
         symbolsByLine[line].push_back(symbol);
+        assertionsByLine[line]; // create entry in assertionsByLine, if none exists yet.
```

Comment:
Are you doing this so the loop below iterates over all lines in the set of assertions and reported results?

Reply:
Actually, I'm not sure if this is actually needed? If you have a symbol in the results that isn't in assertionsByLine, then the second loop will figure that out... right?

Reply:
Before this change, there were two loops:  one on assertionsByLine, and one on symbolsByLine.
After this change, there is only one loop: on assertionsByLine.